### ISSUE TITLE
How to use FlowLimiterCalculator to monitor multiple outputs for the same frame?

### ISSUE THREAD

**Issue by 424778940z** [google/mediapipe#3611]
I'm trying to make a graph that process pose/hands/face/iris, AND being able to switch them on and off seprately.
The problem I got is, there seems no way to monitor multiple outputs with `FlowLimiterCalculator`.
Thus, I cannot make sure all results are from the same frame.

I tried following, but im not sure how to make empty `LandmarkList`  with `ConstantSidePacketCalculator`
And also not sure if `ConcatenateListsCalculator` will work for the trick.
There seems no any AND/OR/NOT logic kind of calculator I can play with.

Could somebody help?

```shell
# CONSTANTS
node {
  calculator: "ConstantSidePacketCalculator"
  output_side_packet: "PACKET:0:empty_landmark"
  node_options: {
    [type.googleapis.com/mediapipe.ConstantSidePacketCalculatorOptions]: {
      packet { LandmarkList : {0} }
    }
  }
}

# MUX
node {
  calculator: "SwitchMuxCalculator"
  input_side_packet: "ENABLE:enable_pose_detection"
  input_stream: "C0__FUNC_INPUT:empty_landmark"
  input_stream: "C1__FUNC_INPUT:pose_landmarks"
  output_stream: "FUNC_INPUT:pose_landmarks_ready"
}
node {
  calculator: "SwitchMuxCalculator"
  input_side_packet: "ENABLE:enable_hands_detection"
  input_stream: "C0__FUNC_INPUT:empty_landmark"
  input_stream: "C1__FUNC_INPUT:left_hand_landmarks"
  output_stream: "FUNC_INPUT:left_hand_landmarks_ready"
}
node {
  calculator: "SwitchMuxCalculator"
  input_side_packet: "ENABLE:enable_hands_detection"
  input_stream: "C0__FUNC_INPUT:empty_landmark"
  input_stream: "C1__FUNC_INPUT:right_hand_landmarks"
  output_stream: "FUNC_INPUT:right_hand_landmarks_ready"
}
node {
  calculator: "SwitchMuxCalculator"
  input_side_packet: "ENABLE:enable_face_detection"
  input_stream: "C0__FUNC_INPUT:empty_landmark"
  input_stream: "C1__FUNC_INPUT:face_landmarks"
  output_stream: "FUNC_INPUT:face_landmarks_ready"
}

node {
  calculator: "ConcatenateListsCalculator"
  input_stream: "pose_landmarks_ready"
  input_stream: "left_hand_landmarks_ready"
  input_stream: "right_hand_landmarks_ready"
  input_stream: "face_landmarks_ready"
  output_stream: "flow_limiter_trigger"
}

node {
  calculator: "FlowLimiterCalculator"
  input_stream: "input_frame"
  input_stream: "FINISHED:flow_limiter_trigger"
  input_stream_info: {
    tag_index: "FINISHED"
    back_edge: true
  }
  output_stream: "throttled_input_frame"
  node_options: {
    [type.googleapis.com/mediapipe.FlowLimiterCalculatorOptions] {
      max_in_flight: 1
      max_in_queue: 1
      # Timeout is disabled (set to 0) as first frame processing can take more
      # than 1 second.
      in_flight_timeout: 0
    }
  }
}
```

**Comment by kuaashish** [comment]
Hi @424778940z, Have you looked at this closed issue #575 and [#575(comment)](https://github.com/google/mediapipe/issues/575#issuecomment-609509826) to use multiple solution in a graph and monitor the output. Thank you!

**Comment by 424778940z** [comment]
> Hi @424778940z, Have you looked at this closed issue #575 and [#575(comment)](https://github.com/google/mediapipe/issues/575#issuecomment-609509826) to use multiple solution in a graph and monitor the output. Thank you!

Thanks for your reply, but seems they are still monitoring a single output, which is not ideal for my case, I want monitor multiple result for the same frame, but seems the Flow Limiter only support a single input

**Comment by kuaashish** [comment]
Hello @424778940z,

We are upgrading the [MediaPipe Legacy Solutions](https://developers.google.com/mediapipe/solutions/guide#legacy) to [new MediaPipe solutions](https://developers.google.com/mediapipe/solutions) However, the libraries, documentation, and source code for all the MediaPipe Legacy Solutions will continue to be available in our [GitHub repository](https://github.com/google/mediapipe) and through library distribution services, such as [Maven](https://maven.google.com/web/index.html?#com.google.mediapipe) and [NPM](https://www.npmjs.com/search?q=%40mediapipe).

You can continue to use those legacy solutions in your applications if you choose. Though, we would request you to check [new MediaPipe solutions](https://developers.google.com/mediapipe/solutions) which can help you more easily build and customise ML solutions for your applications. These new solutions will provide a superset of capabilities available in the legacy solutions. Thank you!
